From: Hilko Bengen <bengen@debian.org>
Date: Mon, 9 Dec 2013 08:58:34 +0100
Subject: Create an empty file that is going to be edited by xgettext

This avoids:

    xgettext: error while opening "libguestfs.pot-t" for reading: No such file or directory
---
 po/Makefile.am | 1 +
 1 file changed, 1 insertion(+)

diff --git a/po/Makefile.am b/po/Makefile.am
index a8343ec..fd69441 100644
--- a/po/Makefile.am
+++ b/po/Makefile.am
@@ -67,6 +67,7 @@ XGETTEXT_ARGS = \
 
 $(DOMAIN).pot: Makefile POTFILES $(POTFILES) POTFILES-pl $(POTFILES_PL) POTFILES-ml $(POTFILES_ML)
 	rm -f $@-t
+	touch $@-t
 if HAVE_OCAML_GETTEXT
 	$(OCAML_GETTEXT) --action extract --extract-pot $@-t $(POTFILES_ML)
 endif
